drivers/gpio/Kconfig.nct38xx

```
@@ -0,0 +1,41 @@
+# NPCX GPIO driver configuration options
+
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_NCT38XX
+	bool "NCT38XX I2C-based GPIO chip"
+	depends on I2C
+	help
+	  Enable driver for NCT38XX I2C-based GPIO chip.
+
+if GPIO_NCT38XX
+
+config GPIO_NCT38XX_INIT_PRIORITY
+	int "Init priority"
+	default 30
+	help
+	  Device driver initialization priority. The priority should be lower
+	  than I2C device.
+
+config GPIO_NCT38XX_PORT_INIT_PRIORITY
+	int "Init priority"
+	default 40
+	help
+	  Device driver initialization priority. The priority should be lower
+	  than I2C & GPIO_NCT38XX_INIT_PRIORITY device.
+
+config GPIO_NCT38XX_INTERRUPT
+	bool "Interrupt enable"
```

As per the guidelines, please remove `enable`:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

Maybe instead write:
```suggestion
	bool "NCT38XX GPIO interrupt"
```

---

Please check in PR #40579.